online_meetings/bluejeans_report.py

```diff
@@ -0,0 +1,186 @@
+import requests
+# standard libraries
+import json
+import logging
+import os
+import sys
+import yaml
+
+# third-party libraries
+import pandas as pd
+import furl
+import time
+import dateparser
+import pytz
+from datetime import datetime, timedelta
+import glob
+
+# importing required modules
+from zipfile import ZipFile
+
+# Initialize settings and globals
+
+logger = logging.getLogger(__name__)
+
+# Default time to sleep when hitting a 429 limit
+DEFAULT_SLEEP_TIME = 10
+
+# read configurations
+try:
+    with open(os.path.join(os.path.dirname(__file__), '../config/env.json')) as env_file:
+        ENV = yaml.safe_load(env_file.read())
+except FileNotFoundError:
+    sys.exit(
+        'Configuration file could not be found; please add env.json to the config directory.')
+
+logging.basicConfig(level=ENV.get('LOG_LEVEL', 'DEBUG'))
+
+url = "https://api.bluejeans.com/oauth2/token?Client"
+
+payload = "{ \"grant_type\": \"client_credentials\", \"client_id\": \"Metrics_BJN\", \"client_secret\": \"" + ENV.get('BLUEJEANS_CLIENT_SECRET') + "\"}"
```

Comment:
You should be able to just define this as a dictionary rather than a string, would save on some quote escaping. If it's an API that needs json encoding the you should still define it as a dict and when posting it call `data=json.dumps(payload)`. Requests also lets you call `json=payload`.

Reply:
Created as new issue